index.js

```diff
@@ -28,6 +32,13 @@ const PREFERRED_WEIGHT = 100;
 const ANNOTATIONS_PATH = 'metadata.annotations';
 const CONTAINER_WAITING_REASON_PATH = 'status.containerStatuses.0.state.waiting.reason';
 
+const DOCKER_ANNOTATIONS = [
+    'screwdriver.cd/dockerEnabled',
+    'screwdriver.cd/dockerCpu',
+    'screwdriver.cd/dockerRam'
```

Comment:
Thank you for contributing! this is going to be very helpful for a lot of ppl! 💃 
Can you put these annotations to the base? https://github.com/screwdriver-cd/executor-base/blob/master/index.js#L7
And you can remove the parseAnnotations function in here

Reply:
https://github.com/screwdriver-cd/executor-base/pull/56/files

Reply:
will have to merge that first and change the dep version in this pr